Formula/kfctl.rb

```diff
@@ -0,0 +1,20 @@
+class Kfctl < Formula
+  desc "Kubeflow CLI tool"
+  homepage "https://www.kubeflow.org"
+  url "https://github.com/kubeflow/kubeflow/releases/download/v0.5.1/kfctl_v0.5.1_darwin.tar.gz"
+  version "0.5.1"
+  sha256 "37790a2bd02a27f94e97c550bc6743a8bdbf69751e78d9e968d018c3bf2bc503"
+  head "https://github.com/kubeflow/kubeflow.git"
+
+  bottle :unneeded
+
+  depends_on "kubernetes-cli"
+
+  def install
+    bin.install "kfctl"
```

Comment:
Unless there's nothing OS or architecture specific, this should be compiling/making/whatever the program.

Reply:
Okay, so I guess this PR needs to wait until Kubeflow actually releases the [source code](https://github.com/kubeflow/kubeflow/tree/master/kfctl)?

Reply:
Yes, it's not even allowed without. https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae